games-emulation/hatari/hatari-2.1.0.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=6
 PYTHON_COMPAT=( python2_7 )
-inherit toolchain-funcs cmake-utils python-single-r1
```

Comment:
This is a major change, so revbump is necessary.

Reply:
Done.